client/src/app/site/services/operator.service.ts

```diff
@@ -588,7 +588,7 @@ export class OperatorService {
             // console.warn(`has perms: Usage outside of meeting!`);
             return false;
         }
-        if (this.isSuperAdmin && !this.activeMeeting.locked_from_inside) {
+        if ((this.isSuperAdmin || this.isOrgaManager) && !this.activeMeeting.locked_from_inside) {
```

Comment:
This check is done quite often. Maybe add an property. Note that you are doing this check also outside of the operator service. So maybe it even makes sense to add that property as public. 

```suggestion
        if (this.canSkipPermissionCheck && !this.activeMeeting.locked_from_inside) {
```